map_type: voxel
origin_z: 0.0
z_resolution: 0.2
z_voxels: 10
unknown_threshold: 9
mark_threshold: 0

transform_tolerance: 0.3
obstacle_range: 2.5
max_obstacle_height: 0.4
raytrace_range: 3.0
footprint: [[-0.53, -0.31], [-0.53, 0.31], [0.53, 0.31], [0.57, 0.0], [0.53, -0.31]]
inflation_radius: 0.15
cost_scaling_factor: 10.0
#unknown as obstacle, when 0, as free.
unknown_cost_value: 255
observation_sources: scan2d nifti_point_cloud
scan2d: {data_type: LaserScan, expected_update_rate: 0.0,
  observation_persistence: 0.0, marking: true, clearing: true, max_obstacle_height: 0.4, min_obstacle_height: 0.08}
scan: {data_type: LaserScan, expected_update_rate: 0.0,
  observation_persistence: 0.0, marking: true, clearing: true, max_obstacle_height: 0.4, min_obstacle_height: 0.08}
nifti_point_cloud: {data_type: PointCloud2, expected_update_rate: 0.0,
  observation_persistence: 0.0, marking: true, clearing: true, max_obstacle_height: 0.4, min_obstacle_height: 0.08}
